README.md

```diff
@@ -2,7 +2,10 @@
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
+##Introduction<br>
+* ResidenceTracker is a command line based desktop application that helps landlords to manage their residences that they rented out.
+* It provides convenience to the  user by providing a centralised overview of all his/her residences and their respective booking/cleaning statuses. 
```

Comment:
@awzhenyi Very minor but there is a typo (double space) between 'the' and 'user'. Otherwise, LGTM!

Reply:
Thanks, done!